apps/website/src/pages/admin/twitch/index.tsx

```diff
@@ -14,8 +15,18 @@ import { ProvideAuth } from "@/components/admin/twitch/ProvideAuth";
 import { ChannelConfig } from "@/components/admin/twitch/ChannelConfig";
 
 export async function getServerSideProps(context: NextPageContext) {
+  const session = await getSession(context);
   const adminProps = await getAdminSSP(context, permissions.manageTwitchApi);
-  return adminProps ? { props: adminProps } : { notFound: true };
+  if (!adminProps) {
+    return {
+      redirect: {
+        destination: session?.user?.id ? "unauthorized" : "/auth/signin",
```

Comment:
Tiny nit, but can we use the same absolute path here too?

```suggestion
        destination: session?.user?.id ? "/admin/unauthorized" : "/auth/signin",
```